api/who/who.proto

```diff
@@ -29,13 +29,20 @@ message PutLocationRequest {
   string s2CellIdToken = 11;
 }
 
+message GetCaseStatsRequest {
+  // ISO_2_CODE
+  string countryCode = 1;
```

Comment:
Atm, we don't need this, we're just giving back the Client's country's stats (and global).